feature-libs/cart/added-to-cart-toast/core/services/added-to-cart-toast.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@angular/core';
+import { Product, ProductService } from '@spartacus/core';
```

Comment:
Could this be more loosely coupled from the product feature?  ( not importing  the product service from the toast feature )
The Product feature will be extracted to a lib and loading it from the toast would load the product lib right from the start.

Reply:
After a bit of working on it, it seems that there are missing pieces of data needed for the cart component. If we want to decouple from the product feature, we would require additional data in the cart event.